styles/style.css

```diff
@@ -0,0 +1,1028 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+font-size: 16px;
+display: flex;
+flex-direction: column;
+ width: 1020px;
+/* position: relative; */
+/* height: 100%; */
+position: absolute;
+left: 50%;
+transform: translateX(-50%);
+
+
+}
+
+
+ 
+ header {
+    background-color: #323232;
+    height: 63px; 
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    display: inline-block;
+   
+    
+} 
+
+header:after {
+content: "";
+width: 0px;
```

Comment:
if you needed 0 values in CSS - just use 0, without units

Reply:
Ok